docs/src/configuration/app/firewall.md

```diff
@@ -85,8 +91,18 @@ firewall:
 
 Requests to port 80 on any IP will be allowed, and requests to 1.2.3.4 on either port 80 or 443 will be allowed, even though the first rule only lists port 443.
 
-## Usage considerations
+## Default Outbound Restrictions Policy
 
-Be aware that many services your application may wish to connect to will be using a domain name that is not on a fixed IP address, or is load-balanced between multiple IP addresses.  You will need to contact the administrator of that service in order to determine the correct IP addresses to allow.
+By default outbound rules have an `allow` policy, which means that if no outbound rules is set, all TCP traffic (except for port 25) is allowed.
+
+The default policy can be changed on a project basis, patching the `outbound_restrictions_default_policy` setting accordingly: `allow | deny`.
+
+```bash
+platform p:curl -X PATCH --project PROJECT_ID /settings -d '{"outbound_restrictions_default_policy": "deny"}'
+```
+
+NOTE: This is an operation permitted only to platform admins.
```

Comment:
If only Platform employees can do so, it should be documented in playbook, not here.  If anything, the public docs should just say "can be set to deny.  File a support ticket and ask the support team to do so."